Error while checking build/O2/fullCI_slc9 for 26b3a499770c1f468cb26457394acdcdf8f2a613 at 2025-07-22 12:34:
```
## sw/BUILD/O2-latest/log
/sw/SOURCES/O2/14377-slc9_x86-64/0/Common/Constants/include/CommonConstants/PhysicsConstants.h:201:18: error: redefinition of 'constexpr const double o2::constants::physics::MassLambda1520'
/sw/SOURCES/O2/14377-slc9_x86-64/0/Common/Constants/include/CommonConstants/PhysicsConstants.h:201:18: error: redefinition of 'constexpr const double o2::constants::physics::MassLambda1520'
/sw/SOURCES/O2/14377-slc9_x86-64/0/Common/Constants/include/CommonConstants/PhysicsConstants.h:201:18: error: redefinition of 'constexpr const double o2::constants::physics::MassLambda1520'
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/14377/26b3a499770c1f468cb26457394acdcdf8f2a613/build_O2_fullCI_slc9/pretty.html).

--- a/Common/Constants/include/CommonConstants/PhysicsConstants.h
+++ b/Common/Constants/include/CommonConstants/PhysicsConstants.h
@@ -89,7 +89,7 @@ enum Pdg {
   kHyperHelium4 = 1010020040,
   kHyperHelium5 = 1010020050,
   kHyperHelium4Sigma = 1110020040,
-  kLambda1520 = 102134
+  kLambda1520_Py = 102134
 };
 
 /// \brief Declarations of masses for additional particles
@@ -151,7 +151,7 @@ constexpr double MassHyperHydrogen4 = 3.922434;
 constexpr double MassHyperHelium4 = 3.921728;
 constexpr double MassHyperHelium5 = 4.839961;
 constexpr double MassHyperHelium4Sigma = 3.995;
-constexpr double MassLambda1520 = 1.5195;
+constexpr double MassLambda1520_Py = 1.5195;
 
 /// \brief Declarations of masses for particles in ROOT PDG_t
 constexpr double MassDown = 0.00467;

--- a/Common/Constants/include/CommonConstants/make_pdg_header.py
+++ b/Common/Constants/include/CommonConstants/make_pdg_header.py
@@ -144,7 +144,7 @@ class Pdg(Enum):
     kHyperHelium4 = 1010020040
     kHyperHelium5 = 1010020050
     kHyperHelium4Sigma = 1110020040
-    kLambda1520 = 102134
+    kLambda1520_Py = 102134
 
 dbPdg = ROOT.o2.O2DatabasePDG
 

--- a/Common/Constants/include/CommonConstants/make_pdg_header.py
+++ b/Common/Constants/include/CommonConstants/make_pdg_header.py
@@ -144,7 +144,7 @@ class Pdg(Enum):
     kHyperHelium4 = 1010020040
     kHyperHelium5 = 1010020050
     kHyperHelium4Sigma = 1110020040
-    kLambda1520_Py = 102134
+    kLambda1520_Py = 102134  # PYTHIA code different from PDG
 
 dbPdg = ROOT.o2.O2DatabasePDG
 

